src/test/java/com/nihongo/EmployeeControllerTest.java

```diff
@@ -133,55 +128,40 @@ void deleteEmployeeTest() throws Exception {
                 .content(objectMapper.writeValueAsString(employeeResponse))
                 .accept("application/json")).andReturn().getResponse();
 
-        Assertions.assertEquals(200, response.getStatus());
+        assertEquals(200, response.getStatus());
 
         String id = response.getContentAsString();
-        Assertions.assertEquals("1",id);
-        //Map map = objectMapper.readValue(response.getContentAsString(), Map.class);
-    }
+        assertEquals("1",id);
 
-    //-===================================================================================================================================
+    }
 
     @Test
     void getEmployeeTest() throws Exception {
         EmployeeResponse employeeResponse = new EmployeeResponse();
-        employeeResponse.setId(Long.parseLong("1"));
+        employeeResponse.setId(1L);
         employeeResponse.setUsername("sidd");
         employeeResponse.setMail("mail");
         employeeResponse.setDob("0-month");
         employeeResponse.setAddress("add");
 
         Employee employee = new Employee();
-        employee.setEmployeeId(Long.parseLong("1"));
+        employee.setEmployeeId(1L);
         employee.setUsername("sidd");
         employee.setMail("mail");
         employee.setDob("0-month");
         employee.setAddress("add");
 
-
-        when(employeeService.getEmployee(any())).thenReturn(List.of(employee));
+        when(employeeService.getEmployee(1L)).thenReturn(List.of(employee));
 
         MockHttpServletResponse response = mockMvc.perform(get("/employees?id=1")
                 .contentType("application/json")
-                .content(objectMapper.writeValueAsString(employeeResponse))
+               // .content(objectMapper.writeValueAsString(employeeResponse))
                 .accept("application/json")).andReturn().getResponse();
 
-        Assertions.assertEquals(200, response.getStatus());
-       // ArrayList map = objectMapper.readValue(response.getContentAsString(), ArrayList.class);
-       // Assertions.assertEquals(1, map.get(1));
+        assertEquals(200, response.getStatus());
 
-        String id = response.getContentAsString();
-        Assertions.assertEquals(1, 1);
-
-    }
+        String jsonResponse = response.getContentAsString();
+        List<Employee> employees = objectMapper.readValue(jsonResponse, new TypeReference<List<Employee>>() {});
 
```

Comment:
where is the assert for validating employess list data?
either you can match id or username like this
assertEquals("1", employees.get(0).getId());
assertEquals("username", employees.get(0).getUsername());

Reply:
done